python/tank/bootstrap/configuration.py

```diff
@@ -31,6 +31,13 @@ def __init__(self, path):
         """
         self._path = path
 
+    @property
```

Comment:
Exposes the path where the configuration will be located. Useful for unit tests.